### ISSUE TITLE
Vitest excludes aren't clearly specified

### ISSUE THREAD

**Issue by cfife-btig** [vitest-dev/vitest#1613]
### Describe the bug

Tl;dr - when running vitest and `vitest.config.ts` hasn't explicitly configured its `exclude`, this is displayed

```
 DEV  v0.17.0 /root/.cache/test

include: **/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}
watch exclude:  **/node_modules/**, **/dist/**

No test files found, exiting with code 1
root@335a94a8766a:~/.cache/test# vitest
```

But really it should show this

```
 DEV  v0.17.0 /root/.cache/test

include: **/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}
watch exclude:  ['**/node_modules/**', '**/dist/**', '**/cypress/**', '**/.{idea,git,cache,output,temp}/**']

No test files found, exiting with code 1
root@335a94a8766a:~/.cache/test# vitest
```

I went down a rabbit hole when my tests were in a `.cache` directory but were failing to be found. A quick documentation tweak probably will save the next sucker a good amount of time.

### Reproduction

Create a .cache folder,

put any test in there i.e. `touch something.test.js`

run vitest. It will fail and claim to not find any test files. but then it doesn't show that the `**/.cache/**` patter is excluded

### System Info

```shell
Probably shouldn't matter but

  System:
    OS: Linux 5.10 Ubuntu 20.04.2 LTS (Focal Fossa)
    CPU: (12) x64 Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz
    Memory: 8.21 GB / 19.55 GB
    Container: Yes
    Shell: 5.0.17 - /bin/bash
  Binaries:
    Node: 16.13.1 - /workspaces/afx/bazel-afx/external/nodejs_linux_amd64/bin/nodejs/bin/node
    Yarn: 1.22.19 - /workspaces/afx/bazel-afx/external/nodejs_linux_amd64/bin/nodejs/bin/yarn
    npm: 8.1.2 - /workspaces/afx/bazel-afx/external/nodejs_linux_amd64/bin/nodejs/bin/npm
```
```


### Used Package Manager

yarn

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitest-dev/vitest/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitest-dev/vitest/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitest.dev/guide/).
- [X] Check that there isn't [already an issue](https://github.com/vitest-dev/vitest/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitest-dev/vitest/discussions) or join our [Discord Chat Server](https://chat.vitest.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by sheremet-va** [comment]
this is "watch exclude", and not "exclude". these are two different things. one will disable hmr for certain files, the other will exclude tests.
